cilium-enterprise-mixin/dashboards/cilium-operator.json

```diff
@@ -186,60 +270,59 @@
           "sort": "none"
         }
       },
-      "pluginVersion": "9.2.0",
+      "pluginVersion": "9.1.5-0100a6a",
       "targets": [
         {
           "datasource": {
             "type": "prometheus",
-            "uid": "${prometheus_datasource}"
+            "uid": "grafanacloud-prom"
```

Comment:
Please ensure the uids are not changed from `${prometheus_datasource}` as it will otherwise not work on generic setups using the templated datasource.
This needs to be done throughout

Reply:
changed all in cilium-operator, however I notice theres still many instances of these throughout the other dashboard files?

Reply:
You're quite right, thought I had caught them earlier but must have used a regex that didn't quite capture all the cases. Will update my PR accordingly